### DISCUSSION TITLE
Object detection on AXIS M4318-PLVE Panoramic Camera: ViewAreas/QuadViews

### DISCUSSION THREAD

**Discussion by AmarRobo** [AxisCommunications/acap-native-sdk-examples#175] [General] [answered]
I am trying the object detection on the panoramic camera following the [object-detection example](https://github.com/AxisCommunications/acap-native-sdk-examples/tree/main/object-detection). The web interface of the camera provides different stream options like overview, panorama,overview,viewarea1 etc. Is it possible to select a view, lets say viewArea1, for the object detection? I could find that the resolution could be changed but can the input stream to the AI model be altered as in the viewAreas?

**Comment by pataxis** [comment, marked as answer]
Hi @amararjun , thanks for your question.

VDO API documentation for [vdo-stream](https://axiscommunications.github.io/acap-documentation/docs/api/src/api/vdostream/html/vdo-stream_8h.html) mentions `channel`, could this be what you are looking for?

In the example the [channel](https://github.com/AxisCommunications/acap-native-sdk-examples/blob/main/object-detection/app/imgprovider.c#L291) is set to 1.

**Reply by AmarRobo** [reply]
@pataxis Thank you! I could select the desired view.
